### ISSUE TITLE
Windows - Incorrect Path Applying Changes From Chat

### ISSUE THREAD

**Issue by avimar** [getcursor/cursor#1542]
1) If you can, please include a screenshot of your problem
When I click "apply" from a chat file, it seems to double the root path:

C:\Users\avi\coding\freeswitch\c:\Users\avi\coding\freeswitch\src\mod\say\mod_say_he\mod_say_he.c
When it's simply: C:\Users\avi\coding\freeswitch\src\mod\say\mod_say_he\mod_say_he.c

![image](https://github.com/user-attachments/assets/7508f9fd-3246-46bc-becf-2d31ab7eec24)

![image](https://github.com/user-attachments/assets/220bdee1-caa5-4ad8-a3ab-6d9317e5e16e)

2) Please include the name of your operating system.
Windows 11 Home
Cursor: 
Version: 0.36.2
VSCode Version: 1.89.1
Commit: 9a6d781540ba2732cf26551c07629e68121c6580
Date: (6 days ago)

3) If you can, steps to reproduce are super helpful

I just used chat and hit "apply"

**Comment by Boscop** [comment]
Yes, this is very annoying indeed, that such a fundamental Cursor feature isn't working on Windows. I ran into it the first time I tried Cursor:

![image](https://github.com/user-attachments/assets/7503190c-6981-4c8a-bdc0-013c92cbffe2)
